language/move-vm/runtime/src/interpreter.rs

```diff
@@ -824,8 +982,10 @@ impl CallStack {
 struct Frame {
     pc: u16,
     locals: Locals,
+    local_tys: Vec<ParanoidType>,
```

Comment:
Locals now carry a type as well.

Reply:
I think these can probably be read directly from `function.locals`